main.py

```diff
@@ -11,32 +11,27 @@ def main() -> None:
 
     for key, value in data.items():
         race_data = value["race"] if value["race"] else None
-        if not Race.objects.filter(name=race_data["name"]).exists():
-            Race.objects.create(
-                name=race_data["name"],
-                description=race_data["description"])
-            for skill in race_data["skills"]:
-                print(skill)
-                Skill.objects.create(
-                    name=skill["name"],
-                    bonus=skill["bonus"],
-                    race=Race.objects.get(name=race_data["name"]))
-
-        guild_data = value["guild"] if value["guild"] else None
-        if value["guild"] is not None:
-            if not Guild.objects.filter(name=guild_data["name"]).exists():
-                Guild.objects.create(
-                    name=guild_data["name"],
-                    description=guild_data["description"])
-
-        if not Player.objects.filter(nickname=key).exists():
-            Player.objects.create(
-                nickname=key,
-                email=value["email"],
-                bio=value["bio"],
-                race=Race.objects.get(name=race_data["name"]),
-                guild=Guild.objects.get(name=guild_data["name"])
-                if value["guild"] else None)
+        race_name, _ = Race.objects.get_or_create(
+            name=race_data["name"],
+            description=race_data["description"])
+
+        skills_data = race_data["skills"]
+        for skill in skills_data:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                bonus=skill["bonus"],
+                race=race_name)
+
+        guild_data = value["guild"]
+        Player.objects.get_or_create(
+            nickname=key,
+            email=value["email"],
+            bio=value["bio"],
+            race=race_name,
+            guild=Guild.objects.get_or_create(
```

Comment:
it's not a good idea to get the guild in Player.objects.get_or_create, move this code above instead of guild_data = value["guild"]